src/view/components/wrappedField.jsx

```diff
@@ -44,7 +44,7 @@ const getNestedValue = (obj, path) => {
 const noop = () => {};
 
 export class DecoratedInput extends React.Component {
-  openDataElementSelector = () => {
+  openDataElementSelector = replace => {
     const {
       field: { onChange, value = "", name }
     } = this.props;
```

Comment:
You can pull `replace` directly off props here instead of passing as it argument if you want.